package.json

```diff
@@ -26,6 +26,7 @@
     "lint:styles": "stylelint \"{src,demo}/**/*.{css,scss}\"",
     "lint:prettier": "prettier --check \"{src,demo}/**/*.{js,jsx,ts,tsx,css,scss}\"",
     "test": "jest",
+    "check-circular-deps": "node scripts/check-circular-deps.js 99",
```

Comment:
I set the threshold to 99 for cyclic dependencies - in the future we should aim for zero :)